### DISCUSSION TITLE
CAN Data Mask / Screen available?

### DISCUSSION THREAD

**Discussion by Retrack90** [collin80/SavvyCAN#852] [Q&A]
Hello everyone. 

I have a CAN Node that sends a lot of messages over one ID and I need the Data Payload only when the first byte is 0x80 and the second byte is 0x15 

the payload is 8 byte long. 

is there a way to mask the rest of the data stream when the befor mentioned conditions are not met. because at the moment I can't really read the incoming data.

**Comment by collin80** [comment]
Well, not entirely right now. You could create a DBC file such that the first byte is used as a multiplexor and that would allow you to sort of filter to a multiplex value of 0x80. And, extended multiplexing is possible where you then use the second byte for a second level of multiplexing. But, all of this will only allow you to selectively interpret just those messages. It won't actually cause any masking of the other messages to occur. This is functionality that could probably be added but there is no prevision for this level of masking right now.

**Comment by bqstony** [comment]
I facing the same problem. To filter only the ID is in my case not the best option. To exclude some values of the first Data Byte would be great.
